### PR TITLE
Improve trial expiry labels

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Changes proposed in this pull request:

- change messages in the trial expiry labels to be more specific.

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/instance_get.go
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/broker/instance_get.go:72]: nit: personally, I would make this configuration option more flexible allowing any subset of subaccounts per environment. I see a benefit in allowing all in dev but restricting to one in prod.

**Review by wozniakjan — DISMISSED** [review]
/lgtm
/approve
/hold

feel free to unhold if extensive configuratbility is not in scope

File: components/kyma-environment-broker/internal/broker/instance_get.go
Comment (szwedm) [reply components/kyma-environment-broker/internal/broker/instance_get.go:72]: I liked the idea of more fexibility in providing more subaccount IDs. However, this requires more changes in the code, which would be removed not too long after it has been merged. If we keep PR as it is now, the subaccount id is read from a environment variable. Since it's impossible to store an array in a environment variable, I would need to provide a serialization/deserialization mechanism for reading subaccount IDs. And we still have to consider limitations of a ConfigMap.

**Review by piotrmiskiewicz — DISMISSED** [review]

File: components/kyma-environment-broker/internal/broker/instance_get.go
Comment (szwedm) [reply components/kyma-environment-broker/internal/broker/instance_get.go]: I changed the condition to support more subaccount IDs.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**
